<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_fio">
          <property name="text">
           <string>Фамилия Имя Отчество</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_pos">
          <property name="text">
           <string>Должность</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Станция техобслуживания компьютеров</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../images/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="admin_page">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QTabWidget" name="admin_tab">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="currentIndex">
                 <number>4</number>
                </property>
                <widget class="QWidget" name="admin_orders_tab">
                 <attribute name="title">
                  <string>Заказы</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="3" column="0">
                   <widget class="QPushButton" name="del_ord_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Удалить запись</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QPushButton" name="save_ord_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Сохранить</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QTableWidget" name="order_table">
                    <attribute name="horizontalHeaderStretchLastSection">
                     <bool>true</bool>
                    </attribute>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="add_ord_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Новый заказ</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QPushButton" name="upd_ord_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Обновить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="admin_warehouse_tab">
                 <attribute name="title">
                  <string>Склад</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_9">
                  <item row="1" column="0">
                   <widget class="QPushButton" name="del_wh_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Удалить запись</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QTableWidget" name="wh_table">
                    <attribute name="horizontalHeaderStretchLastSection">
                     <bool>true</bool>
                    </attribute>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QPushButton" name="save_wh_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Сохранить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="admin_clients_tab">
                 <attribute name="title">
                  <string>Клиенты</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <widget class="QTableWidget" name="client_table">
                    <attribute name="horizontalHeaderStretchLastSection">
                     <bool>true</bool>
                    </attribute>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="del_cl_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Удалить запись</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QPushButton" name="save_cl_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Сохранить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="admin_services_tab">
                 <attribute name="title">
                  <string>Услуги</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QTableWidget" name="service_table">
                    <attribute name="horizontalHeaderStretchLastSection">
                     <bool>true</bool>
                    </attribute>
                    <column>
                     <property name="text">
                      <string>ID Услуги</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Название услуги</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Стоимость, руб.</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="del_serv_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Удалить запись</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QPushButton" name="save_serv_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Сохранить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="admin_history_tab">
                 <attribute name="title">
                  <string>История</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_10">
                  <item row="0" column="0">
                   <widget class="QTableWidget" name="history_table">
                    <attribute name="horizontalHeaderStretchLastSection">
                     <bool>true</bool>
                    </attribute>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_hist">
                    <property name="font">
                     <font>
                      <pointsize>20</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Нет доступа</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="admin_employees_tab">
                 <attribute name="title">
                  <string>Сотрудники</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="1" column="0">
                   <widget class="QPushButton" name="del_empl_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Удалить запись</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QTableWidget" name="empl_table"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QPushButton" name="save_empl_btn">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(238, 130, 238);</string>
                    </property>
                    <property name="text">
                     <string>Сохранить</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_emp">
                    <property name="font">
                     <font>
                      <pointsize>20</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Нет доступа</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="seller_page">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget_2">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="tab">
                 <attribute name="title">
                  <string>Tab 1</string>
                 </attribute>
                </widget>
                <widget class="QWidget" name="tab_2">
                 <attribute name="title">
                  <string>Tab 2</string>
                 </attribute>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="exit_btn">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 130, 238);</string>
            </property>
            <property name="text">
             <string>Выход</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1363</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
